build.sbt

```
@@ -50,6 +50,19 @@ ThisBuild / githubWorkflowBuild     ~= { steps =>
 ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("11"))
 ThisBuild / tlBspCrossProjectPlatforms := Set(JVMPlatform)
 
+ThisBuild / githubWorkflowAddedJobs +=
+  WorkflowJob(
+    id = "coverage",
+    name = s"Generate coverage report (2.13 JVM only)",
+    scalas = List(Scala2),
```

Let's copy the cats-parse setup here (we should port these changes to Skunk too). This way the CI workflow does not need to be re-generated every time there is a Scala version update.

```suggestion
    scalas = Nil,
    sbtStepPreamble = Nil,
```

---

Good point. I've made the necessary recommended changes.